packages/web/src/scss/components/SplitButton/_SplitButton.scss

```
@@ -0,0 +1,25 @@
+// 1. Non-zero values work only when the default Button border radius is a bit smaller than its height. In other words, it doesn't work when Buttons use the full radius.
+
+@use 'sass:map';
+@use '@tokens' as tokens;
+
+.SplitButton {
+    --#{tokens.$css-variable-prefix}segmented-group-gap: calc(
+        -1 * var(--#{tokens.$css-variable-prefix}border-width-100)
+    );
```

I don't see a benefit in using calc and var in this case, 
`--#{tokens.$css-variable-prefix}segmented-group-gap: -#{tokens.$border-width-100};`

---

I agree with skipping the calc and var, I would suggest keeping the `-1 *`, because imho it improves the readability 

```suggestion
    --#{tokens.$css-variable-prefix}split-button-gap: -1 * #{tokens.$border-width-100};
```

---

also, please, can you move this to `theme.scss` file? Either the whole equation or just the border-width token. The same please for the inner-border-radius value (just the 0).

The goal is to have in this file only `tokens.$css-variable-prefix` and everything else `tokens.*` or any other config value have there.